web/src/containers/devices/constants.js

```diff
@@ -73,6 +74,21 @@ export const commandTimeoutUnits = {
   NS: 'ns',
 }
 
+export const devicesProvisionStatuses = {
+  UNINITIALIZED: 'uninitialized',
+  INITIALIZED: 'initialized',
+  PROVISIONING_CREDENTIALS: 'provisioning credentials',
+  PROVISIONED_CREDENTIALS: 'provisioned credentials',
+  PROVISIONING_ACLS: 'provisioning acls',
+  PROVISIONED_ACLS: 'provisioned acls',
+  PROVISIONING_CLOUD: 'provisioning cloud',
+  PROVISIONED_CLOUD: 'provisioned cloud',
+  PROVISIONED: 'provisioned',
+  TRANSIENT_FAILURE: 'transient failure',
+  FAILURE: 'failure',
+}
+export const DEVICE_PROVISION_STATUS_TIMEOUT = 100
```

Comment:
what is the unit ? -> because for some iot devices it can takes around 4-5seconds (for establishing TLS)
generally you can expect for any get/update resource that it takes around 5seconds -> eg at kistler we used 15seconds for any action. (own device, update resource, get resource, create resource, delete resource)